mono/utils/mono-state.c

```diff
@@ -1151,4 +1151,18 @@ mono_crash_dump (const char *jsonFile, MonoStackHash *hashes)
 	return;
 }
 
+static volatile int32_t dump_status;
+
+gboolean
+mono_dump_start (void)
```

Comment:
Need a `mono_dump_start` / `mono_dump_complete` for the case where we have DISABLE_CRASH_REPORTING.